lib/media/track/noisecancellationimpl.ts

```diff
@@ -65,6 +71,14 @@ export class NoiseCancellationImpl implements NoiseCancellation {
     }
     return processedTrack;
   }
+
+  /**
+   * disables noise cancellation permanently.
+   */
+  disablePermanently(): Promise<void> {
```

Comment:
Instead of another public API, what are your thoughts on update `.disable()` and add a boolean value?

```ts
disable(permanently: boolean = false): Promise<void>;
```